interface-definitions/include/ospf/high-low-label-value.xml.i

```diff
@@ -0,0 +1,26 @@
+<!-- include start from ospf/high-low-label-value.xml.i -->
```

Comment:
This PR is for ISIS but this file is for OSPF? It's actually never included, what's the purpose?

Reply:
So, this is something that was odd. After the last PR, I went and I rebased this branch/fork that I have so I can keep working on VyOS. When I went to make my changes I noticed that this file was completely missing. The rebase that I did was on VyOS master. I was very confused too. So I just remade the file again and continued on. It shouldn't have gone missing, but it *seems* to have. I don't know why.